contracts/data/BinaryHeapAddressMock.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.8;
+
+import { BinaryHeap } from './BinaryHeap.sol';
+
+contract BinaryHeapAddressMock {
+    using BinaryHeap for BinaryHeap.AddressHeap;
+
+    BinaryHeap.AddressHeap internal addressHeap;
+
+    function heap() external view returns (address[] memory) {
+        uint256 l = addressHeap.length();
+        address[] memory arr = new address[](l);
+        for (uint256 i = 0; i < l; i++) {
+            arr[i] = addressHeap.at(i);
+        }
+        return arr;
+    }
```

Comment:
Might be better to add a `toArray` function to the library itself, and then use that here.  It wouldn't be gas-efficient, but could be useful in `view` functions.  Also could probably paginate it (though I don't have an opinion about what such a function signature should look like).

Reply:
added in [ca1de73](https://github.com/solidstate-network/solidstate-solidity/pull/134/commits/ca1de730c91abf77147a36748336aeaa0fd5d169)